### ISSUE TITLE
wrong package suggestion on debian to validate passenger and apache installation

### ISSUE THREAD

**Issue by sebastian-schmid** [phusion/passenger#2356]
## Issue report

Question 1: What is the problem?

passenger-config validate-install throws the following message and suggests a package "apache2-threaded-dev" to be installed. But this package does not exist at least on Debian 10 (buster), was not able to find that package for older debian versions too but I am not sure if it existed some time back. Instead you need to install the "apache2-dev" package which contains the required "apxs2" tool.

Expected behavior is that the package "apache2-dev" will be suggested on debian (at least on 10 (buster)).

Can be reproduced by just following the Installation Guide: https://www.phusionpassenger.com/docs/advanced_guides/install_and_upgrade/nginx/install/oss/buster.html

Console output:
```
What would you like to validate?
Use <space> to select.
If the menu doesn't display correctly, press '!'

   ⬢  Passenger itself
 ‣ ⬢  Apache

-------------------------------------------------------------------------

 * Checking whether this Passenger install is in PATH... ✓
 * Checking whether there are no other Passenger installations... ✓
 * Checking whether Apache is installed... ✗

   Unable to validate your Apache installation: more software required
   
   This program requires the apxs2 tool in order to be able to validate your
   Apache installation. This tool is currently not installed. You can solve this
   as follows:
   
   Please install it with apt-get install apache2-threaded-dev
   
   When done, please re-run this program.


Detected 1 error(s), 0 warning(s).
```

Your answer:

**Question 2: Passenger version and integration mode**:

Passenger: 6.0.8 (open source, libapache2-mod-passenger)

**Question 3: OS or Linux distro, platform (including version)**:

Debian 10 (buster)

**Question 4: Passenger installation method**:

Your answer:
[ ] RubyGems + Gemfile
[ ] RubyGems, no Gemfile
[x] Phusion APT repo
[ ] Phusion YUM repo
[ ] OS X Homebrew
[ ] source tarball
[ ] Other, please specify:

**Question 5: Your app's programming language (including any version managers) and framework (including versions)**:
- For example: Ruby 2.3.0, RVM, Rails 5.0.0; Node.js 4.0.0 with Express 4.13.4

Your answer: RVM, Ruby 2.7.3

**Question 6: Are you using a PaaS and/or containerization? If so which one?**
- For example: Heroku, Amazon Container Services, Docker 1.9 with an image based on passenger-docker

Your answer: no

**Question 7: Anything else about your setup that we should know?**

Your answer: I used a new debian 10 VM and installed everything with ansible.

**Comment by CamJN** [comment]
This is due to a bug in the version comparison logic, I've fixed it in 08d9f5b1 and will make a release asap.

**Comment by CamJN** [comment]
Release is done.
